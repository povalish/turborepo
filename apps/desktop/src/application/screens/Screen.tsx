import React from 'react';

import { Button } from 'ui-kit/components/Button';
import { Layout, textClasses } from 'ui-kit/layouts/Layout';

// Screen
//

export const Screen: React.FC = () => (
  <Layout>
    <h1 className={textClasses({ className: 'mb-2' })}>Hello from Electron!</h1>
    <Button type='button'>Lets go!</Button>
  </Layout>
);
