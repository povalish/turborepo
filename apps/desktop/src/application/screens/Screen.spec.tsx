import { render } from '@testing-library/react';
import { Screen } from './Screen';

describe('Screen component', () => {
  it('renders component unchanged', () => {
    const { container } = render(<Screen />);
    expect(container).toMatchSnapshot();
  });
});
