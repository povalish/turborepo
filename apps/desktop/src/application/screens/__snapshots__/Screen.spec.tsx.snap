// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen component renders component unchanged 1`] = `
<div>
  <div
    class="
  w-screen h-screen 
  bg-[#6FD0F3]
  flex flex-col items-center justify-center
  px-4
 
  text-[#333E53] text-3xl text-center
  font-bold
"
  >
    <h1
      class="
  text-[#333E53] text-3xl text-center
  font-bold
 mb-2"
    >
      Hello from Electron!
    </h1>
    <button
      class="
  min-w-[160px] py-1.5 px-2 rounded-lg
  bg-[#FCB07E] border-[#FCB07E] border-2
 
  text-[#333E53] text-xl text-center
  font-bold
"
      type="button"
    >
      Lets go!
    </button>
  </div>
</div>
`;
